addons/l10n_gcc_pos/static/src/overrides/pos_store.js

```diff
@@ -9,6 +9,7 @@ patch(PosStore.prototype, {
                 this.company.country_id?.code
             ),
             gcc_cashier: order?.getCashierName() || this.get_cashier()?.name,
+            show_title: Boolean(order),
```

Comment:
Normally in Js it should be showTitle but i've seen a comment about this no i will not block for that